packages/google_sign_in/README.md

```diff
@@ -0,0 +1,84 @@
+# google_sign_in_tizen
+
+[![pub package](https://img.shields.io/pub/v/google_sign_in_tizen.svg)](https://pub.dev/packages/google_sign_in_tizen)
+
+The Tizen implementation of [`google_sign_in`](https://github.com/flutter/plugins/tree/master_archive/packages/google_sign_in/google_sign_in).
+
+## Usage
+
+This package is not an _endorsed_ implementation of `google_sign_in`. Therefore, you have to include `google_sign_in_tizen` alongside `google_sign_in` as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  google_sign_in: ^5.4.0
+  google_sign_in_tizen: ^0.1.0
+```
+
+For detailed usage on how to use `google_sign_in`, see https://pub.dev/packages/google_sign_in#usage.
+
+## Tizen integration
+
+`google_sign_in_tizen` supports Google Sign-In for [TVs and Limited Input Devices](https://developers.google.com/identity/gsi/web/guides/devices), note that many API scopes are limited for these device types, see [supported scopes](https://developers.google.com/identity/protocols/oauth2/limited-input-device#allowedscopes).
+
+The plugin uses [OAuth 2.0 Device Authorization Grant](https://datatracker.ietf.org/doc/html/rfc8628) (aka device flow) which has a different authorization flow than [Authorization Code Grant with PKCE](https://datatracker.ietf.org/doc/html/rfc7636) used for endorsed platforms (Android, iOS, and web). While every step needed to authorize the app for "authorization code grant with PKCE" happens on a single device, "device flow" requires the user to use a secondary device (PC, tablet, or desktop) to finalize the authorization.
+
+To access Google Sign-In for Tizen, you'll need to make sure to [register your application for TVs and limited input devices](https://developers.google.com/identity/gsi/web/guides/devices). You also need to add some additional code to your app to fully integrate `google_sign_in` on Tizen (explained below).
+
+### Adding OAuth credentials
+
+Unlike "authorization code grant with PKCE", "device flow" requires [client secret](https://developers.google.com/identity/protocols/oauth2/limited-input-device#step-4:-poll-googles-authorization-server) parameter during token request. You must call `setCredentials` function with your client's OAuth credentials before calling `google_sign_in`'s API.
+
+```dart
+import 'package:google_sign_in_tizen/google_sign_in_tizen.dart as tizen'
+
+tizen.setCredentials(
+  clientId: 'YOUR_CLIENT_ID',
+  clientSecret: 'YOUR_CLIENT_SECRET',
+);
+```
+
+:warning: Security concerns
+
+Storing client secret in code is considered bad practice as it exposes [security vulnerabilites](https://datatracker.ietf.org/doc/html/rfc8628#section-5.6). Therefore Google allows a [limited set of scopes](https://developers.google.com/identity/protocols/oauth2/limited-input-device#allowedscopes) that can be requested for device flow. Nevertheless, you can perform extra steps to protect your credentials better.
```

Comment:
You might refine this paragraph ("Therefore ~ better.")

Reply:
How about:

Although Google limits API scopes for Device Flow to protect user's sensitive data, you can perform extra steps to protect your client credentials.